src/components/Teams/team.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Row} from "react-bootstrap";
+
+import "./team.css";
+class Team extends Component {
+	state = {};
+	render() {
+		return (
+			<div className="card">
+				<img className="photo" alt="teams" src={this.props.image} />
+				<div>
+					<h1>{this.props.name}</h1>
+					<h1>{this.props.number}</h1>
+					<Row className="media1">
+						<a href='mailto:{this.props.facebook}' target="_blank" rel="noopener noreferrer">
+							<img className="media" alt="media" src={require("../../resources/commingSoon/facebook.png")} />
```

Comment:
Remove social icons image and use fontawesome icons: https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use.